OpenCombine.podspec

```diff
@@ -0,0 +1,29 @@
+Pod::Spec.new do |spec|
+  spec.name         = "OpenCombine"
+  spec.version      = "0.0.1"
+  spec.summary      = "Open source implementation of Apple's Combine framework for processing values over time."
+
+  spec.description  = <<-DESC
+  Open source implementation of Apple's Combine framework for processing values over time.
+  DESC
+
+  spec.homepage     = "https://github.com/broadwaylamb/OpenCombine/"
+
+  spec.license      = "Apache 2.0"
+
+  spec.author       = "Apple"
+  spec.source       = { :git => "https://github.com/broadwaylamb/OpenCombine", :tag => "#{spec.version}" }
+
+  spec.swift_version = '5.0'
+
+  spec.osx.deployment_target     = '10.9'
+  spec.ios.deployment_target     = '8.0'
+
+  spec.source_files = "Sources/**/*.swift"
```

Comment:
I'm afraid this is not enough. We have several targets in our SPM manifest, and some of them are in C++.

We should probably create subspecs for each target: OpenCombine, OpenCombineDispatch and COpenCombineHelpers (that's the one written in C++).

I haven't used CocoaPods for a while though, so if you have better ideas, feel free to propose them.